### ISSUE TITLE
Reference Doc Github action: Make it easier to see which branch corresponds to each job

### ISSUE THREAD

**Issue by karenzone** [elastic/docs-tools#67]
I'm loving the new github actions to generate plugin docs!  Thank you for investing the time and effort, @jsvd. 

It would be super helpful if there was an easy way to see which branch corresponds to each job. 

<img width="1248" alt="Screen Shot 2022-03-17 at 7 34 56 PM" src="https://user-images.githubusercontent.com/35154725/158910823-a690743e-7df4-4920-9db4-ca9b90de0f04.png">

**Comment by mashhurs** [comment]
My investigations show current feature is not feasible for now. According to [GH variable explanation doc](https://docs.github.com/en/actions/learn-github-actions/environment-variables) seems we can leverage `GITHUB_WORKFLOW` param but it is not settable. Doc does not mention that it is read-only and in case I have tried with sample action workflow and got en error.
There are people who faced similar cases and requested GH to make it writable. Here is [the GH discussion](https://github.com/community/community/discussions/11396) and I also posted a comment that for the worst case scenario to update the description (eg.: Reference Documentation generation 6: Manually run by mashhurs) would satisfy our cases.

Meanwhile I will try to discuss the alternative options like adding branch info to job summary or attaching to job name, etc..
